Revit_Core_Adapter/Revit_Core_Adapter.csproj

```
@@ -201,8 +201,13 @@
     </Reference>
     <Reference Include="System.Windows.Forms" />
     <Reference Include="PresentationCore" />
+    <Reference Include="UI_Engine">
+      <HintPath>C:\ProgramData\BHoM\Assemblies\UI_Engine.dll</HintPath>
+    </Reference>
+    <Reference Include="UI_oM">
+      <HintPath>C:\ProgramData\BHoM\Assemblies\UI_oM.dll</HintPath>
+    </Reference>
```

Would be good to do this just like what's already been done for other assemblies:
```suggestion
    <Reference Include="UI_Engine">
      <HintPath>C:\ProgramData\BHoM\Assemblies\UI_Engine.dll</HintPath>
      <Private>False</Private>
      <SpecificVersion>False</SpecificVersion>
    </Reference>
    <Reference Include="UI_oM">
      <HintPath>C:\ProgramData\BHoM\Assemblies\UI_oM.dll</HintPath>
      <Private>False</Private>
      <SpecificVersion>False</SpecificVersion>
    </Reference>
```

---

The bot will not let you pass without the attributes I think - so not only good but necessary to add them 👍